server/src/test/resources/META-INF/services/org.elasticsearch.plugins.spi.NamedXContentProvider

```diff
@@ -1 +1 @@
-org.elasticsearch.plugins.spi.NamedXContentProviderTests$TestNamedXContentProvider
\ No newline at end of file
+org.elasticsearch.plugins.spi.NamedXContentProviderTests$TestNamedXContentProvider
```

Comment:
this is an unrelated change?

Reply:
Yep sorry, this was part of reverting Igor's changes that missed a newline